### DISCUSSION TITLE
paralel atama

### DISCUSSION THREAD

**Discussion by Ferbman** [19/classroom#4] [Ruby] [answered]
a1  = gets.to_i
b1  = gets.to_i
c1  = gets.to_i

a1 , b1 , c1 = gets.to_i, gets.to_i, gets.to_i

1. yazımı 2. yazım  olarak yazmanın yolu var mı?

kodu çalıştırdıktan sonra konsolda a1 değeri ender b1 değeri enter şeklinde girmek istemiyorum  a1 b1 c1 olarak tek seferde girmek istiyorum

(a1 , b1 , c1 = gets.to_i, gets.to_i, gets.to_i  .   formatında olmak zorunda değil değerleri yan yana girebileceğim şekilde yazmak istiyorum

**Comment by MelihArtar** [comment]
ilkk = gets.chomp
ilk = ilkk.split()
ilka = ilk[0].to_f
ilkb = ilk[1].to_f
ilkc = ilk[2].to_f

şeklinde yazarsan sırasıyla a1 b1 c1 kodlarını alırsın.

**Reply by Ferbman** [reply]
dizi kullanmak istemiyorum başka yol biliyor musun?

**Reply by MelihArtar** [reply]
ben sadece dizi oluşturarak yapmayı biliyorum başka aklıma gelmedi

**Reply by roktas** [reply]
```ruby
a, b ,c = gets.chomp.split.map(&:to_i)
```

Aradığın cevabı sunmak için yazdım.  Ama şunu unutma, bir programın dış dünya ile (kullanıcılar veya dış servisler) kontak kurduğu noktalar kritiktir.  Bu kontak noktalarında "kısayollar"a pek dalma.  Neden?  Hata denetimi yapman gerekiyor ve bu pek "kısa" bir iş değil.

**Reply by Ferbman** [reply]
teşekkür ederim. İnputlara dikkat edeceğim.

**Comment by roktas** [comment, marked as answer]
```ruby
a, b ,c = gets.chomp.split.map(&:to_i)
```

Aradığın cevabı sunmak için yazdım.  Ama şunu unutma, bir programın dış dünya ile (kullanıcılar veya dış servisler) kontak kurduğu noktalar kritiktir.  Bu kontak noktalarında "kısayollar"a pek dalma.  Neden?  Hata denetimi yapman gerekiyor ve bu pek "kısa" bir iş değil.
